polymetis/polymetis/python/polymetis/robot_client/executable_robot_client.py

```diff
@@ -66,6 +66,8 @@ def run(self):
 
                 # Find path to executable
                 path_to_exec = which(self.executable_cfg.exec)
+                log.warn("Executing " + path_to_exec)
```

Comment:
```suggestion
                log.info("Executing " + path_to_exec)
```

Reply:
Oh, this was just a debug line - I'll drop it entirely, thanks for catching it.